tests/components/mqtt/test_common.py

```diff
@@ -119,6 +119,30 @@ async def help_setup_component(
     return mqtt_mock
 
 
+def help_custom_config(
+    domain: str,
+    default_config: ConfigType,
+    option_update: ConfigType | tuple[ConfigType,],
+) -> ConfigType:
+    """Tweak a default config for parametrization.
+
+    Returns a custom config to be used as parametrization for for hass_config,
+    based on a supplied default_config and updated with option_update(s).
+    For each option_update an instance is added to the config.
+    """
+    config: ConfigType = copy.deepcopy(default_config)
+    if isinstance(option_update, tuple):
```

Comment:
I don't think we need to support both, just require that it's always an iterable of entity configs?